packages/tools/devtools/devtools-view/src/components/Menu.tsx

```diff
@@ -34,12 +57,15 @@ const useMenuStyles = makeStyles({
 		"flexDirection": "column",
 		"height": "100%",
 		"overflowY": "auto",
-		"minWidth": "150px",
+		"minWidth": "200px",
 		// Ensures the last div/component is anchored to the bottom.
 		"> :last-child": {
 			marginTop: "auto",
 			marginBottom: "15px",
 		},
+		"> div > div": {
+			marginBottom: "4px",
+		},
```

Comment:
This could use some documentation. I would also recommend using a classname for the target rather than using this syntax, if possible.